### PR TITLE
sysvar: add description in INSTANCE sysvars

### LINKED ISSUE #10842: 文档中关于 INSTANCE 作用域变量的描述不够清楚
虽然在 [文档](https://docs.pingcap.com/zh/tidb/stable/system-variables#%E7%B3%BB%E7%BB%9F%E5%8F%98%E9%87%8F) 开头有关于 “是否持久化到集群” 的说明，但很难被注意到

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Although the current document has a clear description of INSTANCE scope variables at the beginning, users may still get confused when they reach the specified item by searching the document.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

close #10842
- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]

**Review by zimulala — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ mysql> SELECT * FROM t1;
 ### `ddl_slow_threshold`
 
 - 作用域：GLOBAL
-- 是否持久化到集群：否
+- 是否持久化到集群：否，仅作用于当前 TiDB 实例
```
Comment (TomShawn) [inline system-variables.md:156]: 可以都改成
```suggestion
- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
```

File: system-variables.md [hunk 2]
```diff
@@ -238,14 +238,14 @@ mysql> SELECT * FROM t1;
 ### `plugin_dir`
 
 - 作用域：GLOBAL
-- 是否持久化到集群：否
+- 是否持久化到集群：否，仅作用于当前 TiDB 实例
```
Comment (TomShawn) [inline system-variables.md:241]:
```suggestion
- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
```

File: system-variables.md [hunk 3]
```diff
@@ -238,14 +238,14 @@ mysql> SELECT * FROM t1;
...
 - 默认值：""
 - 指定加载插件的目录。
 
 ### `plugin_load`
 
 - 作用域：GLOBAL
-- 是否持久化到集群：否
+- 是否持久化到集群：否，仅作用于当前 TiDB 实例
```
Comment (TomShawn) [inline system-variables.md:248]:
```suggestion
- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
```

File: system-variables.md [hunk 4]
```diff
@@ -473,7 +473,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 ### `tidb_check_mb4_value_in_utf8`
 
 - 作用域：GLOBAL
-- 是否持久化到集群：否
+- 是否持久化到集群：否，仅作用于当前 TiDB 实例
```
Comment (TomShawn) [inline system-variables.md:476]:
```suggestion
- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
```

File: system-variables.md [hunk 5]
```diff
@@ -747,7 +747,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 ### `tidb_enable_collect_execution_info`
 
 - 作用域：GLOBAL
-- 是否持久化到集群：否
+- 是否持久化到集群：否，仅作用于当前 TiDB 实例
```
Comment (TomShawn) [inline system-variables.md:750]:
```suggestion
- 是否持久化到集群：否，仅作用于当前连接的 TiDB 实例
```

File: system-variables.md [hunk 6]
```diff
@@ -929,7 +929,7 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 ### `tidb_enable_slow_log`
 
 - 作用域：GLOBAL
-- 是否持久化到集群：否
+- 是否持久化到集群：否，仅作用于当前 TiDB 实例
```
Comment (TomShawn) [inline system-variables.md:932]: ditto for the rest

File: system-variables.md (hunk 1 above)
Comment (CbcWestwolf) [reply system-variables.md]: Done

**Pull request merged**